x-pack/plugins/security_solution/public/common/components/event_details/table/investigate_in_timeline_button.tsx

```diff
@@ -46,14 +56,14 @@ export const InvestigateInTimelineButton: React.FunctionComponent<{
       dispatch(
         sourcererActions.setSelectedDataView({
           id: SourcererScopeName.timeline,
-          selectedDataViewId: 'security-solution-default',
-          selectedPatterns: ['.alerts-security.alerts-default'],
+          selectedDataViewId: defaultDataView.id,
+          selectedPatterns: [signalIndexName || ''],
         })
       );
       // Unlock the time range from the global time range
       dispatch(inputsActions.removeLinkTo([InputsModelId.timeline, InputsModelId.global]));
     }
-  }, [dispatch, clearTimeline, dataProviders]);
+  }, [dataProviders, clearTimeline, dispatch, defaultDataView.id, signalIndexName]);
```

Comment:
@elastic/security-threat-hunting-investigations reviewer, lookie here! We cannot hard code data view ids since they are case sensitive.